user_docs/en/changes.md

```diff
@@ -4,6 +4,9 @@
 
 ### Important notes
 
+* Users are warned that support for Microsoft Speech API version 4 synthesizers is planned for removal in NVDA 2026.1.
```

Comment:
This wording seems to describe a specific UX, namely the dialog box even if it is not explicitly mentioned. This would be OK for a normal change log item. But it is not consistent with past important notes.

Writing something as follows seems more suitable IMO:

```suggestion
* The support for Microsoft Speech API version 4 synthesizers is planned for removal in NVDA 2026.1.
```